README.md

```diff
@@ -1 +1,25 @@
-# mysqld_exporter (UNDER CONSTRUCTION)
+# mysqld_exporter
+
+Exporter for MySQL server metrics http://prometheus.io/
+
+## Building and running
+
+    make
+    ./mysqld_exporter <flags>
+
+## Configuration
+The configuration is in JSON. An example with all possible options:
+```
+{
+   "config" : {
+      "mysql_connection" : "login:password@host/dbname"
+   }
+}
+```
+Name     | Description
```

Comment:
It's probably best to link off to https://github.com/go-sql-driver/mysql#dsn-data-source-name rather than duplicating their docs.